rmagick.gemspec

```diff
@@ -29,6 +29,8 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= #{Magick::MIN_RUBY_VERSION}"
   s.requirements << "ImageMagick #{Magick::MIN_IM_VERSION} or later"
 
+  s.add_runtime_dependency 'native-package-installer'
```

Comment:
@Watson1978 I wonder if we should do this without introducing a 3rd party dependency. Looking at their source, `native-package-manager` checks for [a `pacman`](https://github.com/ruby-gnome2/native-package-installer/blob/master/lib/native-package-installer/platform/arch-linux.rb#L23) and [a `brew`](https://github.com/ruby-gnome2/native-package-installer/blob/master/lib/native-package-installer/platform/homebrew.rb#L23) command. What if instead we use something like `find_executable('pacman')` and `find_executable('brew')`?

Reply:
It is a library created by a familiar Ruby committer.

If we use the library properly, it will be able to install ImageMagick automatically with `gem install rmagick` command when ImageMagick is not installed on system.

like

https://github.com/apache/arrow/blob/8b152b4e628b0df45464201a78bda9c0305d77fa/ruby/red-arrow/ext/arrow/extconf.rb#L35-L39

https://github.com/ruby-gnome2/ruby-gnome2/blob/74132c157113359b32826ad640a22d40840baeb8/glib2/lib/mkmf-gnome2.rb#L440-L456

I think it will be able to reduce user errors at installation.

Reply:
Hmm, I should add the library when introducing that feature.
I will fix this PR.

Reply:
OK, fixed :)